src/views/payment-sheet-views/paypal-view.js

```diff
@@ -61,7 +61,11 @@ PayPalView.prototype._initialize = function () {
       onError: reportError
     };
 
-    paypal.Button.render(paypalCheckoutConfig, '[data-braintree-id="paypal-button"]').then(function () {
+    if (merchantConfiguration.locale) {
```

Comment:
Do we care about the case where they have a different locale in their top level config and their paypal config?